helmclient.go

```diff
@@ -76,13 +73,13 @@ func New(config Config) (*Client, error) {
 }
 
 // DeleteRelease uninstalls a chart given its release name.
-func (c *Client) DeleteRelease(ctx context.Context, releaseName string, options ...helmclient.DeleteOption) error {
+func (c *Client) DeleteRelease(ctx context.Context, releaseName string) error {
```

Comment:
For Helm 2 we set `helm.DeletePurge(true)` but this option was removed from Helm 3.

https://github.com/giantswarm/chart-operator/blob/86901f94d21f395f540102afa91be92ac7607ee0/service/controller/chart/v1/resource/release/delete.go#L21